test/e2e/playwright.config.ts

```diff
@@ -13,6 +13,7 @@ export default defineConfig({
   expect: {
     timeout: 30000,
   },
+  timeout: 60000, //Set Test timeout to 1 minute
```

Comment:
default timeout of 30 seconds was not enough for the operations to complete in workflow agents